frontend/www/js/omegaup/components/course/Statistics.vue

```diff
@@ -123,28 +169,97 @@ export default class Statistics extends Vue {
     }
     return maxPoints;
   }
-  get maxVariance() {
-    let maxVariance = 0;
-    for (const variance of this.getStatistic('variance')) {
-      if (variance > maxVariance) maxVariance = variance;
+  getMaxStat(statistic: 'variance' | 'avg_runs') {
+    let max = 0;
+    for (const stat of this.getStatistic(statistic)) {
+      if (stat > max) max = stat;
+    }
+    return max;
+  }
+  get problemIndices() {
+    const indices: {
+      [assignmentAlias: string]: { [problemAlias: string]: number };
+    } = {};
+    this.problemStats.forEach((problem, index) => {
+      if (!indices.hasOwnProperty(problem.assignment_alias))
+        indices[problem.assignment_alias] = {};
+      indices[problem.assignment_alias][problem.problem_alias] = index;
+    });
+    return indices;
+  }
+  get runsPerAssignment() {
+    const problems: string[] = this.problems;
+    let runSum: number[] = new Array(problems.length).fill(0);
+    for (const stat of this.verdicts) {
+      runSum[this.problemIndices[stat.assignment_alias][stat.problem_alias]] +=
+        stat.runs;
+    }
+    return runSum;
+  }
+  get verdictList() {
```

Comment:
same as with `problemIndices`: if it's not being used in Vue, it's probably better to remove the getter:

```suggestion
  getVerdictList(): string[] {
```